tldw_Server_API/app/core/MCP_unified/tests/test_http_validation_bounds.py

```
@@ -24,7 +24,7 @@ def _setup_env():
     os.environ["MCP_JWT_SECRET"] = "x" * 64
     os.environ["MCP_API_KEY_SALT"] = "s" * 64
     os.environ["MCP_ENABLE_MEDIA_MODULE"] = "false"
-    os.environ["MCP_MODULES_CONFIG"] = "/tmp/mcp_modules_empty.yaml"
+    os.environ["MCP_MODULES_CONFIG"] = "/tmp/mcp_modules_empty.yaml"  # nosec B108
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/mcp_modules_empty.yaml"

(S108)